Java/AutoML/automl-cms-api/src/main/java/com/automl/cms/api/controller/MLDatasetController.java

```diff
@@ -0,0 +1,35 @@
+package com.automl.cms.api.controller;
+
+import static com.automl.cms.api.model.Result.ok;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.HEAD;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.automl.component.dao.MLDatasetDAO;
+import com.automl.component.model.MLDataset;
+
+/**
+ * @author wangqingwei
+ * Created on 2022-08-10
+ */
+@RestController()
+@RequestMapping("/dataset/")
```

Comment:
In openapi.yaml, the URL definition for dataset is like '/namespaces/{namespace}/datasets', these two are out of sync.